cmd/deploy/command_configuration.go

```diff
@@ -46,6 +46,12 @@ func setDeployOptionsValuesFromManifest(ctx context.Context, deployOptions *Opti
 		deployOptions.Manifest.Namespace = okteto.Context().Namespace
 	}
 
+	remote := utils.LoadBoolean(constants.OktetoDeployRemote)
```

Comment:
The env var should be `OKTETO_FORCE_REMOTE`, `constants.OktetoDeployRemote` = `OKTETO_DEPLOY_REMOTE` which means if the current okteto command is executing on remote  or locally